espresso/key-manager/key_manager.go

```diff
@@ -188,17 +195,24 @@ func (k *EspressoKeyManager) PrepareRegisterService(getAttestationFunc func([]by
 		if err != nil {
 			return nil, nil, fmt.Errorf("nitro signing failed: %w", err)
 		}
+		if k.espressoNitroAttesVerifierClient == nil {
+			return nil, nil, errors.New("attestation verifier client is not initialized")
+		}
```

Comment:
return error if url is empty instead. I think it will be clearer, otherwise people would get confused why this is not initialized.